Mjml.Net/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Mjml.Net.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string? SuffixCssClasses(this string classAttributeValue, string? suffix)
```

Comment:
If there is any way to do this at certain parts allocation free, I'd be keen to know!

Reply:
I don't think so.

Reply:
Only with a lot of changes.